app/models/history.rb

```diff
@@ -344,9 +346,9 @@ def custom_details(fields, patient_identifiers)
     history_details[:user_defined_id_statelocal] = patient_identifiers[:user_defined_id_statelocal]
     history_details[:user_defined_id_cdc] = patient_identifiers[:user_defined_id_cdc]
     history_details[:user_defined_id_nndss] = patient_identifiers[:user_defined_id_nndss]
-    history_details[:created_by] = created_by || '' if fields.include?(:created_by)
+    history_details[:created_by] = remove_formula_start(created_by) || '' if fields.include?(:created_by)
```

Comment:
Good idea, emails definitely can start with `=`.

Reply:
My approach was to scrub any string that didn't have a validation that enforced a format.